vnext/ReactUWP/Views/TextInputViewManager.cpp

```diff
@@ -16,10 +16,25 @@
 
 #include <IReactInstance.h>
 
+#ifdef USE_WINUI3
+namespace winrt {
+namespace Microsoft {
```

Comment:
use nested namespaces?

Reply:
@rectified95 thanks - apparently that's possible now as of C++17 ('twasn't before)